### ISSUE TITLE
Several abandoned transactions during Lendex NFT sale

### ISSUE THREAD

**Issue by gmanredmond** [soonlabs/soonaverse-dao#476]
I purchased the lendex NFT , had several abandoned transactions,
I would lock then signing took very long and I would get the invalid sig message or NFT unavailable message. 

 Once I finally got one to work , the final screen to confirm my transaction went thru and the exit checkout did not present itself. I had to go to my profile and look to see if the NFT was there.  Also I think for NFT sales we should probably not allow animation or mp4 play during that time, I think that was messing with page load times.

**Comment by adamkundrat** [comment]
Can you be more specific? How many times you tried and what exactly were the messages?

Did you purchase from the list (collection page, or marketplace list) or did you go into an individual NFT and hit buy button?

**Comment by adamkundrat** [comment]
This might be related to https://app.zenhub.com/workspaces/public-master-61a51a3547b2090011dce2d2/issues/soonlabs/soonaverse/488

**Comment by gmanredmond** [comment]
I believe i tried it 4 or 5 times.  Went to the Ledex settler collection, opened it scrolled until I found an available one to with an active buy button. 
then it would proceed to > the accept and lock, which I did.> Then waited for the signing process to take place which took a long time. One time it just vanished, I then had to go thru the same process over again, settler collection>scroll to find available nft to buy>accept and lock> wait for signing, then got wither invalid signature or NFT unavailable. repeated this process I thnk 4or 5 time, maybe more. Until it worked.  Hope that's enough info, let me know.
